critiquebrainz/db/__init__.py

```diff
@@ -2,7 +2,7 @@
 from sqlalchemy.pool import NullPool
 
 # This value must be incremented after schema changes on exported tables!
-SCHEMA_VERSION = 1
+SCHEMA_VERSION = 6
```

Comment:
I thought of updating the `db.SCHEMA_VERSION` to `6` as the previous value of `model.__version__` was `6`. I have changed it back to `1`.